swap_meet/item.py

```diff
@@ -1,2 +1,26 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        id = int(uuid.uuid4()) if not id else id
```

Comment:
What would happen on this line, if someone wanted to make initialize an object with an `id` of `0`?